lib/modules/xPostLinks.js

```diff
@@ -0,0 +1,66 @@
+addModule('xPostLinks', function(module, moduleID) {
+	module.moduleName = 'X-post Links';
+	module.category = 'Posts';
+	module.description = 'Create links to x-posted subreddits in the taglines.';
+	module.include = [
+		/^https?:\/\/([a-z]+)\.reddit\.com/i
```

Comment:
That's cool but it also needs to include the /user/ pages. What's the keyword for that, or better how could i possibly find out myself?